packages/component-base/src/element-mixin.js

```diff
@@ -78,3 +78,7 @@ export const ElementMixin = (superClass) =>
       }
     }
   };
+
+export function defineCustomElement(CustomElement) {
+  customElements.define(CustomElement.is, CustomElement);
```

Comment:
Consider extracting this to separate file. Not all the components we use extend `ElementMixin` - in particular, it isn't used by internal elements such as overlays etc.

Reply:
Is there a better existing place or should it be a new file?=

Reply:
I would prefer a new file e.g. `define.js`

Reply:
Moved to `define.js`